src/main/resources/config/application-prod.yml

```diff
@@ -22,10 +22,10 @@ spring:
     datasource:
         type: com.zaxxer.hikari.HikariDataSource
         driverClassName: org.postgresql.Driver
-        url: jdbc:postgresql://localhost:5432/managementportal
```

Comment:
better not change the database name. there are other configurations that need to be changed accordingly.

Reply:
Indeed, i was testing the production profile locally and accidentally committed the changes. Changes to this file have been reverted.